app/views/application/_browsing_index.html.erb

```diff
@@ -1,6 +1,7 @@
 <div class="container">
-<% cache(action: "index", action_suffix: "obituaries_index") do %>
+
   <% @obituaries.each do |obit| %>
```

Comment:
hoping that this associates the cache with each obit object, so when it sees a new one it knows to update the cache